lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -911,7 +911,7 @@ private function getCascadeMappings(SimpleXMLElement $cascadeElement)
     protected function loadMappingFile($file)
     {
         $result = [];
-        $xmlElement = simplexml_load_file($file);
+        $xmlElement = simplexml_load_string(file_get_contents($file));
```

Comment:
> Is there any situation where [`simplexml_load_file()`](http://php.net/manual/en/function.simplexml-load-file.php) is not returning `false` with a not found file?

Indeed, that's what a test would verify, but I see no tests.